.drone.yml

```diff
@@ -0,0 +1,67 @@
+kind: pipeline
+name: test
+
+steps:
+- name: flake8
+  image: alpine/flake8:3.7.8
+  commands:
+  - flake8
+
+- name: pytest
+  image: vmck/vagrant-vmck:0.2.1
+  commands:
+  - export VMCK_URL=http://$VMCK_IP:$VMCK_PORT
+  - export VMCK_NAME="drone $DRONE_REPO:$DRONE_BRANCH#$DRONE_BUILD_NUMBER"
+  - ./ci/deploy-acs-interface.sh
```

Comment:
I think you can move the code from `deploy-acs-interface.sh` to the test itself, it doesn't encapsulate much useful logic. And please move the `Vagrantfile` to the top level of the repo. It's a bad idea to keep it hidden, and it should be reusable by people who want to work on the project, so they don't have to install it locally.